rpsl-parser/proto/rpsl.pb.go

```diff
@@ -26,7 +18,7 @@ var _ = math.Inf
 // is compatible with the proto package it is being compiled against.
 // A compilation error at this line likely means your copy of the
 // proto package needs to be updated.
-const _ = proto.ProtoPackageIsVersion2 // please upgrade the proto package
+const _ = proto.ProtoPackageIsVersion3 // please upgrade the proto package
```

Comment:
why did the generated pb.go change?
it looks like maybe the .proto file changed from proto2 -> proto3 requirements?

Reply:
no that didn't change. I merely reran protoc and the generator itself is a later version